keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/probabilistic/GaussianVertex.java

```diff
@@ -108,21 +106,15 @@ public double logProb(DoubleTensor value) {
 
     @Override
     public LogProbGraph logProbGraph() {
-        final LogProbGraph.DoublePlaceholderVertex xPlaceholder = new LogProbGraph.DoublePlaceholderVertex(this.getShape());
-        final LogProbGraph.DoublePlaceholderVertex muPlaceholder = new LogProbGraph.DoublePlaceholderVertex(mu.getShape());
-        final LogProbGraph.DoublePlaceholderVertex sigmaPlaceholder = new LogProbGraph.DoublePlaceholderVertex(sigma.getShape());
-
-        final DoubleVertex lnSigma = sigmaPlaceholder.log();
-        final DoubleVertex xMinusMuSquared = xPlaceholder.minus(muPlaceholder).pow(2);
-        final DoubleVertex xMinusMuSquaredOver2Variance = xMinusMuSquared.div(sigmaPlaceholder.pow(2).times(2.0));
-
-        final DoubleVertex logProbOutput = xMinusMuSquaredOver2Variance.plus(lnSigma).plus(LN_SQRT_2PI).unaryMinus().sum();
+        final DoublePlaceholderVertex xPlaceholder = new DoublePlaceholderVertex(this.getShape());
+        final DoublePlaceholderVertex muPlaceholder = new DoublePlaceholderVertex(mu.getShape());
+        final DoublePlaceholderVertex sigmaPlaceholder = new DoublePlaceholderVertex(sigma.getShape());
 
         return LogProbGraph.builder()
             .input(this, xPlaceholder)
             .input(mu, muPlaceholder)
             .input(sigma, sigmaPlaceholder)
-            .logProbOutput(logProbOutput)
+            .logProbOutput(Gaussian.logProbOutput(xPlaceholder, muPlaceholder, sigmaPlaceholder).sum())
```

Comment:
why is there a `.sum()` on the end?

Reply:
So it looks the same as the way implemented it in the distribution/vertex